schemas/groups/propulsion.json

```diff
@@ -64,6 +41,12 @@
       "units": "Hz"
     },
 
+    "maxRevolutions": {
```

Comment:
This is probably my confusion. What happens if I subscribe to the parent path `vessels.foo.propulsion` e.g.
```json
"subscribe": [
    {
      "path": "vessels.foo.propulsion",
      "period": 1000,
      "format": "delta",
      "policy": "ideal",
      "minPeriod": 200
    }
]
```
Should I receive:
 * one update for the `propulsion` path with the full value?
 * one update for the `propulsion` path with a sparse value containing children that changed?
 * one update containing each source event that affected a value under the `propulsion` path, with each value affected?